### DISCUSSION TITLE
Document Caching Issue in Remix

### DISCUSSION THREAD

**Discussion by nischalJovian** [remix-run/remix#8432] [Q&A] [answered]
We added caching header, in one of the routes of our Remix app. Even after adding the `max-age` inside the `Cache Control` header, we are getting 200 response in the Network tab. 

We are trying to enable document caching on one of our frequently visited routes and followed this blog for achieving the same: 
https://sergiodxa.com/articles/loader-vs-route-cache-headers-in-remix

<img width="548" alt="Screenshot 2024-01-04 at 7 32 00 PM" src="https://github.com/remix-run/remix/assets/107476975/7c89cb27-3d5a-43d3-9266-a3c874ca41bf">

**Comment by sergiodxa** [comment, marked as answer]
Are you reloading the document? The browser ignores the cache when you do that, instead open a new tab, open the devtools, and then access to your URL by typing it in the browser. You should now see the document being cached.

**Reply by nischalJovian** [reply]
Yes, we were reloading the document. After following the steps you suggested, we are seeing the document being cached.
<img width="623" alt="Screenshot 2024-01-05 at 10 13 59 AM" src="https://github.com/remix-run/remix/assets/107476975/5eb9a3e5-a0cb-4647-82d3-e31194b108d5">
